### ISSUE TITLE
Building on the work of colleagues

### ISSUE THREAD

**Issue by ajburls** [dwyl/bestevidence#243]
A colleague, Dr Guylene Thierault, from McGill University has a project called EBMPICO which provides a structure for creating critically appraised topics that has overlap with one of the next functionalities we were proposing to add to BestEvidence. The source code is available free in Source forge and she says she would be delighted if we were to build on it. Could this be something that would be helpful to Dwyl for the next stage of development of the app, either to use directly or to assist in the design of the next functionality?

Web site: http://larip.uqo.ca/ebmpico/src/index.php?l=en
(not perfect)

Mobile version : http://larip.uqo.ca/mobilepico/#/welcome
(in development)

Source forge: https://sourceforge.net/projects/ebmpico/

**Comment by markwilliamfirth** [comment]
@ajburls please could you assign a priority label to this issue? Can you confirm that the requirements here are to spend 30 minutes researching this application in order to understand whether any learnings from it might be beneficial for the future of the project?

**Comment by ajburls** [comment]
Yes, I just want to know whether the code could be of use to us. Is it the same language etc.? I am not proposing that we add in this functionality in this sprint.

**Comment by iteles** [comment]
Having taken a quick look at EBMPICO, I can note that:
+ It is written in a different language (PHP) which means we need to do some thinking around how this might work with BE
+ There is no documentation - http://larip.uqo.ca/ebmpico/src/services/documentation/manual.php#RechercheArticle - so we'll have to wade through the code itself to understand it
+ All of the current bug tracker is all in French - https://sourceforge.net/p/ebmpico/bugs/?source=navbar - so it's not straightforward to understand what the already reported issues are with it.

All of this tells me that we're not going to get any useful answers with 30 mins of time unfortunately, bumping this up by 2 hours (which I still think is a bit too low).

**Comment by ajburls** [comment]
@iteles I suggest that we don't use our time on this now given your observations, I will lower the priority. I think this may simply be best used later when we want to add our own similar functionality to inform our own design.

**Comment by iteles** [comment]
@ajburls Thanks for confirming. Shall we remove it from the Sprint 5 milestone altogether (i.e. the next possible block of work)?

**Comment by ajburls** [comment]
@iteles Yes, I think we should park it until we are proposing to add structured notes and CAT maker functionality
